.dockerignore

```diff
@@ -17,8 +17,6 @@
 # under the License.
 #
 
-polaris-service/logs
-polaris-service/build
-polaris-core/build
+logs
```

Comment:
is this correct? shouldn't it be dropwizard/service/logs ?
similarly for the `*/build` folders

Reply:
If I add e.g. `dropwizard/service/build` IntelliJ warns me: 

> 'dropwizard/service/build' is covered by 'build'

So, I went for simplicity :-)

Reply:
plain `logs` applies at any nesting level, AFAIK